curriculum/challenges/english/03-front-end-libraries/bootstrap/use-a-span-to-target-inline-elements.md

```diff
@@ -22,7 +22,9 @@ Using a `span` element, nest the word `love` inside the `p` element that current
 
 Here's how you would do this for the `p` element that has the text `Top 3 things cats hate`:
 
-`<p>Top 3 things cats <span class="text-danger">hate:</span></p>`
+```html
+<p>Top 3 things cats <span class="text-danger">hate:</span></p>
+```html
```

Comment:
You have an extra `html` that should not be here.
````suggestion
```
````

Reply:
Tip in the file tab on GitHub you can batch all these suggestions together in one single commit.

Reply:
@LaurentLabine You have still not resolved the issue above. Also, you have still not made a change to the file I referenced in my first comment.

Reply:
Hey, Yeah I' still working on it, sorry!  I am not sure how to manage my current pull request and additional changes that must be added... Should I close this one and re-submit a new one with everything sorted out?

Reply:
I just made the changes on my local branch and was actually wondering how to proceed next wether I could add the missing changes to my current PR, make a new one for the missing file and html correction or whether I should re do it all together closing the initial request we've been working on...

Reply:
Hey there,

If you have changes prepared locally, you should be able to commit them and `git push` up to your remote repository - this PR would then automatically update. 

For the changes Randy has suggested with the GitHub UI here, like Sem mentioned you can switch to the file view and commit them in a batch through the UI - do note that you should do this after you've pushed up your local changes, to avoid conflicts.

Reply:
Hey!  Yeah it got me confused but that is what I understood happened...  My understanding is that it's been integrated and everything about this PR has been dealt with now.. Right?

Reply:
I am still seeing @RandellDawson's suggestions as outstanding.

Reply:
I believe the changes are in and got one approval and the system is waiting for a second one in order to merge the changes?

I find it pretty confusing but I did make the changes and from what I can see the modifications have been integrated in the PR?

Two of the changes I edited and uploaded locally to my branch which I believe as you brought up have been automatically taken into account in the initial PR and the other ones I merged them from the recommendations that were made during the review so that might be why I still have some outstanding comments left hanging?

Reply:
Just realised I had the option to set two changes to resolved... Is it better now?

Reply:
I see a commit you made that resolved a couple of changes, yes - but there are some that were not in that commit as well.